scripts/parse-criterion-output.awk

```diff
@@ -38,16 +38,18 @@
 #
 # To output records like:
 # ```
-# { "throughput": "32.852 MiB/s",  "change": "none",  "throughput_change": "-1.0800%",  "name": "transforms/transforms",  "time_change": "+1.0918%",  "time": "31.932 ms"}
-# { "throughput": null,  "change": "none",  "throughput_change": null,  "name": "complex/complex",  "time_change": "+0.8758%",  "time": "1.8391 s"}
+# { "throughput": "32.852 MiB/s",  "change": "none",  "throughput_change": "-1.0800%",  "name": "transforms/transforms",  "time_change": "+1.0918%",  "time": "31.932 ms", "p": "0.0"}
+# { "throughput": null,  "change": "none",  "throughput_change": null,  "name": "complex/complex",  "time_change": "+0.8758%",  "time": "1.8391 s", "p": "0,2"}
 # ```
 ###
 
 BEGIN {
   # match time:   [1.8272 s 1.8391 s 1.8498 s]
   measurement_regex = "\\[(.+ .+) (.+ .+) (.+ .+)\\]"
-  # match change: [+0.1481% +0.8758% +1.5410%] (p = 0.01 < 0.05)
-  change_regex = "\\[(.+) (.+) (.+)\\]"
+  # match time change: [+0.1481% +0.8758% +1.5410%] (p = 0.01 < 0.05)
+  time_change_regex = "\\[(.+) (.+) (.+)\\] \\(p = (.+) [<>] .+\\)"
+  # match thrpt change: [+0.1481% +0.8758% +1.5410%]
+  thrpt_change_regex = "\\[(.+) (.+) (.+)\\]"
```

Comment:
Should these `.+` all be `\S+`? This assumes some preferred AWK actually has `\S`. gawk does FWIW

Reply:
:+1: `\S+` would be better.